src/parser/jobs/blm/modules/Procs.tsx

```diff
@@ -12,6 +12,7 @@ import React from 'react'
 
 declare module 'event' {
 	interface FieldsTargeted {
+		/** BLM specific logic. Do not use. Please remove ASAP. */
```

Comment:
@AkairyuGestalter 
![image](https://user-images.githubusercontent.com/534235/120320421-822b1580-c325-11eb-8545-9566daaf2c98.png)

left a message in `#dev-caster`, feel free to drop a line, I feel there should be a way to model this without polluting the event structure